tests/site_vars/l2tp.yml

```diff
@@ -0,0 +1,13 @@
+---
+# This site config only enables L2TP
+
+vpn_clients: 5
```

Comment:
Nit pick, but as it is now l2tp doesn't do much with vpn clients, can be set to 1 for this case